membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -56,7 +68,7 @@ class AttributeController extends Controller with LazyLogging {
         request.touchpoint.attrService.get(id).map {
           case Some(attrs) =>
             onSuccess(attrs).withHeaders(
-              "X-Gu-Membership-Tier" -> attrs.Tier,
+              "X-Gu-Membership-Tier" -> attrs.Tier.get,
```

Comment:
There is an extra error here. If the user is only a contributor we should return 404 in order to have backwards compatibility.